node/lib/task.ts

```diff
@@ -511,7 +511,11 @@ export function stats(path: string): FsStats {
  * @returns   boolean 
  */
 export function exist(path: string): boolean {
-    return path && fs.existsSync(path);
+    try {
+        return path && fs.statSync(path) != null;
+    } catch (err) {
+        return false;
```

Comment:
Seems like this works for "right" for checking whether file exists

>>sudo touch test.txt
>>sudo chmod 000 test.txt
>>node test.js test.txt  //returns true the current user is not root
>> ls /etc/usr/bin
ls: cannot access /etc/usr/bin: No such file or directory
>> node test.js /etc/usr/bin //return true